build/dependencies.props

```diff
@@ -3,9 +3,10 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
   <PropertyGroup Label="Package Versions">
+  
     <BenchmarkDotNetPackageVersion>0.10.13</BenchmarkDotNetPackageVersion>
     <InternalAspNetCoreSdkPackageVersion>2.1.0-preview3-17003</InternalAspNetCoreSdkPackageVersion>
```

Comment:
I rolled back this part of the change because it breaks us by whining about symbols packages for our build package. We can solve that some other time.